src/dao/sqlUtils.ts

```diff
@@ -135,9 +122,13 @@ const set = (obj: any, path: any, value: any) => {
   if (!arrayPath.length) return obj
 
   // eslint-disable-next-line no-return-assign
-  arrayPath.slice(0, -1).reduce((a: any, c: any, i: any) => (Object(a[c]) === a[c]
-    ? a[c]
-    : a[c] = Math.abs(arrayPath[i++]) >> 0 === +arrayPath[i++]
+  arrayPath.slice(0, -1).reduce((
+    acc: any,
```

Comment:
Doesn't 'acc' needs to be an object?

Reply:
I put any because the object can be "a: { b: { } }" to avoid the type error I set as any